packages/expo-location/src/Location.ts

```diff
@@ -159,8 +171,39 @@ export async function enableNetworkProviderAsync(): Promise<void> {
 }
 
 export async function getCurrentPositionAsync(
-  options: LocationOptions = {}
+  options: LocationOptions = {},
+  methodOptions: LocationMethod = { type: LocationMethodType.Fast }
 ): Promise<LocationData> {
+  console.log(methodOptions);
+  if (methodOptions.type === LocationMethodType.Fast) {
+    console.log("fast");
+    return new Promise<LocationData>(async (resolve, reject) => {
```

Comment:
Please take a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function 
You do not have to explicit wrap function body into `Promise`, because the result from `async` function is implicitly wrapped in `Promise` 😉

Reply:
Ok, but I need to get resolve and reject function. Could I get them without wrapping in `Promise`?